packages/E2ETest/app/AccessibilityTestPage.tsx

```diff
@@ -3,57 +3,78 @@
-                </Text>
-                <Text style={{ color: 'green' }} onPress={() => setPressedCount(pressedCount + 1)} accessible={true} accessibilityLiveRegion="polite">Pressed {pressedCount} times</Text>
-            </View>
-            <View>
-                <MyView
-                    accessible={true}
-                    accessibilityLabel='AccessibilityAction'
-                    accessibilityRole='CheckBox'
-                    accessibilityStates={['checked', 'expanded']}
-                    accessibilityActions={[
-                        { name: 'toggle', label: 'toggle' },
-                        { name: 'invoke', label: 'invoke' },
-                        { name: 'expand', label: 'expand' },
-                        { name: 'collapse', label: 'collapseIt' },
-                    ]}
-                    onAccessibilityAction={(event: { nativeEvent: { actionName: any; }; }) => {
-                        switch (event.nativeEvent.actionName) {
-                            case 'toggle':
-                                setAccessibilityAction('toggle action success');
-                                break;
-                            case 'invoke':
-                                setAccessibilityAction('invoke action success');
-                                break;
-                            case 'expand':
-                                setAccessibilityAction('expand action success');
-                                break;
-                            case 'collapseIt':
-                                setAccessibilityAction('collapseIt action success');
-                                break;
-                        }
-                    }}
-                >
-                    <Text>accessibilityAction:{accessibilityAction}</Text>
-                </MyView>
-            </View>
-        </View>)
-}
\ No newline at end of file
+  return (
+    <View>
+      <View>
+        <Text
+          accessible={true}
+          accessibilityLiveRegion="polite"
+          style={{ fontWeight: 'bold' }}
+        >
+          I'm bold
+          <Text
+            style={{ color: 'red' }}
+            onPress={() => setPressedCountNested(pressedCountNested + 1)}
+            accessible={true}
+            accessibilityLiveRegion="polite"
+          >
+            Pressed {pressedCountNested} times
+          </Text>
+        </Text>
+        <Text
+          style={{ color: 'green' }}
+          onPress={() => setPressedCount(pressedCount + 1)}
+          accessible={true}
+          accessibilityLiveRegion="polite"
+        >
+          Pressed {pressedCount} times
+        </Text>
+      </View>
+      <View>
+        <MyView
+          accessible={true}
+          accessibilityLabel="AccessibilityAction"
+          accessibilityRole="CheckBox"
+          accessibilityState={{ checked: true, expanded: true }}
```

Comment:
@asklar are we not tree-dumping this? We were previously setting some acc properties that we removed support for, but fixing this doesn't seem to change E2ETest results.

Reply:
The only test pages we tree dump today are the ControlStyles and Image pages.
You can find the list of masters here: [masters](https://github.com/microsoft/react-native-windows/tree/master/packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/masters) 
Feel free to add more though.